infrastructure/instances.tf

```diff
@@ -43,21 +108,48 @@ resource "aws_instance" "data_refinery_worker_1" {
   }
```

Comment:
It might be useful to add the IP address of this as an `output`.

Reply:
:+1: